### ISSUE TITLE
Is it possible to hide the line when no function signature is displayed?

### ISSUE THREAD

**Issue by QiangF** [tuhdo/semantic-stickyfunc-enhance#4]
Hello Sir,
I have a small screen, how can I hide that line?
Thank you!

**Comment by tuhdo** [comment]
Hi,

which line do you want to hide? You can simply turn off with `M-x semantic-stickyfunc-mode`.

**Comment by QiangF** [comment]
I mean the line that displays the function head. I want to keep semantic-stickyfunc-mode, but the package always takes up a line no matter if there is a function name or not.
Thank you!

**Comment by blaenk** [comment]
Yeah this would be cool if it were possible. To phrase it another way, if the point is not inside of a function then it should not show the bar at the top.

**Comment by Martinsos** [comment]
I would also love to see this feature! Actually, right now the bar does not take space - code is still visible through it. What we actually need is for the bar to become "transparent" when not showing a tag/function.
@tuhdo do you have any advice on how this could be done? Btw., I love your blog posts! Great source of emacs ideas.
